lib/util.js

```diff
@@ -304,7 +304,11 @@ function formatValue(ctx, value, recurseTimes) {
       return ctx.stylize(RegExp.prototype.toString.call(value), 'regexp');
     }
     if (isDate(value)) {
-      return ctx.stylize(Date.prototype.toISOString.call(value), 'date');
+      if (value.toString() === 'Invalid Date') {
```

Comment:
Works for me.
On Apr 30, 2016 3:05 AM, "Ben Noordhuis" <notifications@github.com> wrote:

> In lib/util.js
> <https://github.com/nodejs/node/pull/6469#discussion_r61667758>:
>
> > @@ -304,7 +304,11 @@ function formatValue(ctx, value, recurseTimes) {
> >        return ctx.stylize(RegExp.prototype.toString.call(value), 'regexp');
> >      }
> >      if (isDate(value)) {
> > -      return ctx.stylize(Date.prototype.toISOString.call(value), 'date');
> > +      if (value.toString() === 'Invalid Date') {
>
> Insofar that it matters, the way V8 implements the Date object, getTime()
> is faster than valueOf() is faster than toString(). The first one is a
> field lookup, the others are calls into the runtime, with toString()
> polluting the date cache and creating a new string every time.
>
> I'd write it as Number.isNaN(value.getTime()) or Object.is(value.getTime(),
> NaN) to avoid keeping around an extra object, though.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/6469/files/b23061713ea4c52d1280ca5f5d015aa92a62d529#r61667758>
>